app/src/main/res/values-lt/strings.xml

```diff
@@ -0,0 +1,275 @@
+    <string name="edit_profile">Redaguoti paskyrą</string>
+    <string name="edit_your_profile">Redaguokite savo paskyrą</string>
+    <string name="appearance_settings_group_title">Išvaizda</string>
+    <string name="mode">Režimas</string>
+    <string name="colour">Spalva</string>
+    <string name="email">El. paštas</string>
+    <string name="current_week_progress">Dabartinės savaitės eiga</string>
+    <string name="weekly_progress">Savaitės pažanga</string>
+    <string name="your_hydration_statistics">Jūsų hidratacijos statistika</string>
+    <string name="average_consumption">Vidutinis paros suvartojimas</string>
+    <string name="this_week">Šią savaitę</string>
+    <string name="this_month">Šį mėnesį</string>
+    <string name="your_progress_for_this_week">Jūsų pažanga šią savaitę</string>
+    <string name="your_progress_for_the_last_12_weeks">Jūsų pažanga per pastarąsias 12 savaičių</string>
+    <string name="inventory">Inventorius</string>
+    <string name="levels">Lygiai</string>
+    <string name="global">Pasaulinis lyderių sąrašas</string>
+    <string name="goal_creation">Sukurti tikslą</string>
+    <string name="walking">Vaikščiojimas</string>
+    <string name="daily">Kasdien</string>
+    <string name="weekly">Kas savaitę</string>
+    <string name="add_goal">Pridėti tikslą</string>
+    <string name="todays_goals">Šiandienos tikslai</string>
+    <string name="weekly_goals">Savaitės tikslai</string>
+    <string name="create_goal">Sukurti tikslą</string>
+    <string name="period">Laikotarpis</string>
+    <string name="exercise">Pratimas</string>
+    <string name="create">Sukurti</string>
+    <string name="select_an_option">Pasirinkite parinktį</string>
+    <string name="hydration_logs">Hidratacijos žurnalai</string>
+    <string name="hold_drink_button_message">Norėdami gerti, paspauskite ir palaikykite mygtuką</string>
+    <string name="dismiss_snackbar">Atsisakyti užkandžių juostos</string>
+    <string name="are_you_sure">Ar tu tuo tikras\?</string>
+    <string name="sell_for">Parduoti</string>
+    <string name="are_you_sure_you_want_to_sell">Ar tikrai norite parduoti</string>
+    <string name="buy_for">Pirkti</string>
+    <string name="are_you_sure_you_want_to_buy">Ar tikrai norite pirkti</string>
+    <string name="has_been_added_to_your_inventory">buvo įtrauktas į jūsų inventorių</string>
+    <string name="has_been_sold">buvo parduotas</string>
+    <string name="hydration_notification_progress">Šios dienos pažanga:</string>
+    <string name="today__progress_liters">Šiandienos pažanga yra %1$s/%2$s litrų</string>
+    <string name="default_notification_channel_id">Numatytas</string>
+    <string name="you">Tu</string>
+    <string name="progress_tab_title">PROGRESAS</string>
+    <string name="activities_tab_title">VEIKLA</string>
+    <string name="global_leaderboard_tab_title">LYGŲ LENTELĖ</string>
+    <string name="current_month_progress">Einamojo mėnesio eiga</string>
```

Comment:
"LYGŲ LENTELĖ" is misleading and could be replaced with a more accurate translation. Maybe something like "LYDERIŲ LENTELĖ"